crates/ruff_python_formatter/tests/snapshots/black_compatibility@comments6.py.snap

```diff
@@ -183,10 +183,7 @@ aaaaaaaaaaaaa, bbbbbbbbb = map(list, map(itertools.chain.from_iterable, zip(*ite
  
 -AAAAAAAAAAAAA = [AAAAAAAAAAAAA] + SHARED_AAAAAAAAAAAAA + USER_AAAAAAAAAAAAA + AAAAAAAAAAAAA  # type: ignore
 +AAAAAAAAAAAAA = (
-+    [AAAAAAAAAAAAA]
-+    + SHARED_AAAAAAAAAAAAA
-+    + USER_AAAAAAAAAAAAA
-+    + AAAAAAAAAAAAA
++    [AAAAAAAAAAAAA] + SHARED_AAAAAAAAAAAAA + USER_AAAAAAAAAAAAA + AAAAAAAAAAAAA
```

Comment:
Black keeps the line breaks as is because of the type ignore comment. It formats the expression the same as Ruff does now when removing/changing the comment